_data/harnesses/finding-distinct/kstreams.yml

```diff
@@ -0,0 +1,162 @@
+dev:
+  steps:
+    - title: Initialize the project
+      content:
+        - action: execute
+          file: tutorial-steps/dev/init.sh
+          render:
+            file: tutorials/finding-distinct/kstreams/markup/dev/init.adoc
+
+    - title: Get Confluent Platform
+      content:
+        - change_directory: merge-streams
+          action: make_file
+          file: docker-compose.yml
+          render:
+            file: tutorials/finding-distinct/kstreams/markup/dev/make-docker-compose.adoc
+
+        - action: execute_async
+          file: tutorial-steps/dev/docker-compose-up.sh
+          render:
+            file: tutorials/finding-distinct/kstreams/markup/dev/start-compose.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/wait-for-containers.sh
+          render:
+            skip: true
+    - title: Configure the project
+      content:
+        - change_directory: filter-events
+          action: make_file
+          file: build.gradle
+          render:
+            file: tutorials/finding-distinct/kstreams/markup/dev/make-build-file.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/gradle-wrapper.sh
+          render:
+            file: tutorials/finding-distinct/kstreams/markup/dev/make-gradle-wrapper.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/make-configuration-dir.sh
+          render:
+            file: tutorials/finding-distinct/kstreams/markup/dev/make-config-dir.adoc
+
+        - action: make_file
+          file: configuration/dev.properties
+          render:
+            file: tutorials/finding-distinct/kstreams/markup/dev/make-dev-file.adoc
+            
+    - title: Create a schema for the events
+      content:
+        - action: execute
+          file: tutorial-steps/dev/make-avro-dir.sh
+          render:
+            file: tutorials/finding-distinct/kstreams/markup/dev/make-avro-dir.adoc
+
+        - action: make_file
+          file: src/main/avro/click.avsc
+          render:
+            file: tutorials/finding-distinct/kstreams/markup/dev/make-avro-schema.adoc
+
+        - action: execute
+          file: tutorial-steps/dev/build-project.sh
+          render:
+            file: tutorials/filtering/kstreams/markup/dev/build-project.adoc
```

Comment:
`filtering` -> `distinct-events`

Also it looks like you'll need to create both `build-project.sh` and `build-project.adoc` within the `distinct-events` tree.